ci.md

```diff
@@ -0,0 +1,201 @@
+      stable branches, and nightlies for `main` branches.
+    * Library being tested is compiled using CMake and Make.
+* `<library>-ci-pr_any-homebrew-amd64`: Build and test on macOS
+    * Dependencies are installed from homebrew.
+    * Library being tested is compiled using CMake and Make.
+* `<library>-ci-pr_any-windows*-amd64`: Build and test on Windows
+    * External dependencies are installed from vcpkg
+    * Library being tested and its Ignition dependencies are compiled using colcon.
+
+Pull requests to stable branches also trigger:
+
+* `<library>-abichecker-any_to_any-ubuntu_auto-amd64`: Run ABI checker
+
+### GitHub Actions
+
+GitHub actions run on GitHub's servers. Currently, it builds Linux for all
+libraries, and macOS for some of them.
+
+Every commit to any branch will trigger:
+
+* `Ubuntu CI / Ubuntu Bionic CI (push)`: Build, test and run linters on Ubuntu Bionic.
+    * Builds are run inside Docker containers.
+    * By default, dependencies are installed from stable debian binaries for
+      stable branches, and nightlies for `main` branches.
+    * Library being tested is compiled using CMake and Make.
+    * Test coverage is computed and uploaded to Codecov.
+* `Ubuntu CI / Ubuntu Focal CI (push)`: Build, test and run linters on Ubuntu Focal.
+    * Builds are run inside Docker containers.
+    * By default, dependencies are installed from stable debian binaries for
+      stable branches, and nightlies for `main` branches.
+    * Library being tested is compiled using CMake and Make.
+
+Every pull request will also trigger the exact same builds, but with the
+`pull_request` suffix. Note that pull requests from forks won't have the
+`push` checks, while pull requests from the official repositories will have
+both `pull_request` and `push`.
+
+Other checks run on GitHub Actions:
+
+* `DCO`: Checks that all commits are signed correctly for the
+    [Developer Certificate of Origin](https://developercertificate.org/).
+* `PR Collection Labeler`: Adds collection labels (i.e. Blueprint, Citadel...)
+    to the pull request automatically according to the target branch.
+* `Ticket opened`: Adds the pull request to the [Core development board](https://github.com/orgs/ignitionrobotics/projects/3).
+* `codecov/*`: Checks that the test coverage hasn't been reduced.
+
+## Required checks
+
+Some checks are marked as `Required`. This means the pull request can't
+be merged unless that check is green.
+
+Ideally, all checks would be required. But known warnings, test failures and
+flaky tests often prevent us from marking them as required. Until these are
+addressed, some checks will need to be checked by hand when they fail, to see
+if those failures are unexpected. See
+[this issue](https://github.com/ignition-tooling/release-tools/issues/398) for
+what checks are required for each library.
+
+## Interpreting results
+
+On the GitHub UI, checks can be:
+
+* ✅: Passed, all good!
```

Comment:
✅ should be ✔️
Just kidding..........

Reply:
Ohhh interesting. On Ubuntu Chrome, the former is white and latter green. On Android, the former is green and latter black 🤷‍♀️

Reply:
I was annoyed by the same thing :sweat_smile: Gotta ask the Unicode Consortium to be more specific about check colors, those render differently according to the platform :grimacing: 

![image](https://user-images.githubusercontent.com/5751272/117331916-1354bc80-ae4c-11eb-8c3c-20499aa13f92.png)